### DISCUSSION TITLE
When indenting a text block (like a quote) and outputting to pdf,  there is no linebreak, so text continues out to the left of the page. looks really ugly

### DISCUSSION THREAD

**Discussion by emilBeBri** [quarto-dev/quarto-cli#4694] [Q&A] [answered]
As per the description - I almost feel like this should be a bug report, but  I'm using: 

Quartio v. 1.2.335

So not the newest version. Because deno that quarto depends upon gives issues when updating quarto on Arch-based distros using the AUR, I haven't updated it (well I tried and spend half a day trying to fix it and then giving up and rolling back). 

So if someone having the newest version could verify this is still an issue - if it *isn't*, I guess I'll have to find a way to update.

And if it is *still* an issue, how do you fix it? It seems almost unbelievable that something so basic is not working? It works with html output.

The same happens when rendering to pdf using rmarkdown.  So it makes me think that there is something I'm not understanding properly here. Can  anyone shed a light on this for me? Much appreciated.

```


---
title: "Application for skill course"
subtitle: "Writing workshop"
date: "`r Sys.Date()`"

---

# Course description


    DAY 1: Within the first hour the most important writing tools have been introduced. And then we write! There are four intense writing slots before dinner. We keep the energy high with body & brain exercises, a walk-n-talk and by sharing experiences. And you can always ask for coaching if you are stalled. The evening workshop is entertaining and relaxed.

    DAY 2: The morning energy is utilised to get writing done on your own text. Then we have a workshop on ‘storytelling in academic writing’ which often helps clarify the story in your text. There are altogether five writing slots. But energy is managed carefully with a long lunch break, yoga, meditation and optional massage treatments.





```

**Comment by cderv** [comment, marked as answer]
[BlockQuote syntax](https://quarto.org/docs/authoring/markdown-basics.html#other-blocks) is usually use for quotes and with this the tex will be wrapped 
````
---
title: "Application for skill course"
subtitle: "Writing workshop"
date: "`r Sys.Date()`"
---

# Course description


> DAY 1: Within the first hour the most important writing tools have been introduced. And then we write! There are four intense writing slots before dinner. We keep the energy high with body & brain exercises, a walk-n-talk and by sharing experiences. And you can always ask for coaching if you are stalled. The evening workshop is entertaining and relaxed.

> DAY 2: The morning energy is utilised to get writing done on your own text. Then we have a workshop on ‘storytelling in academic writing’ which often helps clarify the story in your text. There are altogether five writing slots. But energy is managed carefully with a long lunch break, yoga, meditation and optional massage treatments.
````

When using 4 space indentation you are using [Verbatim Code block](https://pandoc.org/MANUAL.html#verbatim-code-blocks). This is usually used for code, and by default they do not wrap and keep line length of the code. LaTeX can be configure to help wrap the code but not easy by experience. 

Is not using blockquote a choice ?

**Reply by emilBeBri** [reply]
Oh you're right I completely forgot about the difference between those two!  Thank you.
